src/celmath/geomutil.h

```diff
@@ -15,35 +15,58 @@
 #include <Eigen/Core>
 #include <Eigen/Geometry>
 
+#include <celcompat/numbers.h>
 #include "mathlib.h"
 
 namespace celmath
 {
 
+template<class T> const inline Eigen::Quaternion<T> XRot90{ static_cast<T>(1) / celestia::numbers::sqrt2_v<T>,
+                                                            static_cast<T>(1) / celestia::numbers::sqrt2_v<T>,
+                                                            0,
+                                                            0 };
+
+template<class T> const inline Eigen::Quaternion<T> XRot180{ 0, 1, 0, 0 };
+
+template<class T> const inline Eigen::Quaternion<T> XRot270{ static_cast<T>(1) / celestia::numbers::sqrt2_v<T>,
```

Comment:
Needs a better name: this is a -90° rotation (or conjugate of 90° rotation), not a 270° rotation (sign difference in quaternion representation)